destral/testing.py

```diff
@@ -158,6 +158,66 @@ def test_access_rules(self):
                     ', '.join(no_access)
             ))
 
+    def test_translate_modules(self):
+        """
+        Test translated strings in the module using the .po and .pot files
+        """
+
+        from os.path import join, isdir
+        from tools import trans_export
+        from cStringIO import StringIO
+        from utils import compare_pofiles
+        mod_path = join(self.openerp.config['addons_path'], self.config['module'])
+        trad_path = join(mod_path, 'i18n')
+        if not isdir(trad_path):
+            logger.warning(
+                'Module %s has no translations', self.config['module']
+            )
+            return
+
+        logger.info(
+            'Checking translations for module %s', self.config['module']
+        )
+        with Transaction().start(self.database) as txn:
+            cursor = txn.cursor
+
+            # Generate POT data from loaded strings
+            trans_data = StringIO()
+            trans_export(
+                self.config['testing_langs'][0], self.config['module'],
```

Comment:
@ecarreras I'd check it for each language, but all POT files are identical, isn't it?